persistence/sql/migratest/fixtures/recovery_code/8f75f5d9-9cb4-4848-9a73-9344f686f8a6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": "8f75f5d9-9cb4-4848-9a73-9344f686f8a6",
+  "recovery_address": null,
```

Comment:
The `code` is missing, I assume that is on purpose?

Reply:
Yes, same for the `recovery_token` files. I guess that's because it uses `json.MarshalIndent` and `Code` is ignored in that. See https://github.com/jonas-jonas/kratos/blob/feat/replaceMagicLinksWithCode/selfservice/strategy/code/code_recovery.go#L35

Reply:
👍